examples/nl-core-SNAQScore-01.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-SNAQScore-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-SNAQScore"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="8881000146104"/>
+         <display value="SNAQ"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <effectiveDateTime value="2013-01-27"/>
+   <valueInteger value="4"/>
+   <note>
+      <text value="Grotere mate van ondervoeding t.o.v. vorig meetmoment"/>
```

Comment:
Missing a dot at the end of the sentence

Reply:
Done (also in SNAQrc and SNAQ65+)!